components/private_channel/client_private_channel.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PRIVATE_CHANNEL_CLIENT_PRIVATE_CHANNEL_H_
+#define BRAVE_COMPONENTS_PRIVATE_CHANNEL_CLIENT_PRIVATE_CHANNEL_H_
+
+#include <string>
+#include "brave/components/private_channel/rust/ffi/src/private_channel.h"
+
+struct ChallengeArtefacts {
```

Comment:
this should all be in `private_channel` namespace along with everything else in components/private_channel

Reply:
I'm also not really following why this file exists. Why isn't this all part of private_channel?

Reply:
- [x] refactor `client_private_channel.*` and add it to `private_channel` namespace